packages/bits/demo/src/components/demo/wizard-v2/wizard-custom-layout/wizard-custom.component.less

```diff
@@ -10,24 +10,32 @@
     .nui-wizard-step-header {
         border: @nui-line-default solid;
         border-radius: 5px;
-
         .setCssVariable(border-color, nui-color-line-default);
         .setCssVariable(background, nui-color-bg-content);
-
+        flex-direction: row-reverse;
+        justify-content: flex-end;
         padding: 0 8px;
     }
 
-    .nui-wizard-step-header-selected {
+    .nui-wizard-step-header--selected {
         .setCssVariable(background-color, nui-color-active);
 
         color: @nui-color-text-light;
         border-color: transparent;
 
-        svg path {
-            fill: @nui-color-text-inverse;
+        .nui-wizard-step-label{
+            .setCssVariable(color, nui-color-text-inverse)
+        }
+
+        .nui-wizard-step-header__icon--selected svg circle, .nui-wizard-step-header__icon--selected svg path {
+            .setCssVariable(fill, nui-color-text-inverse)
```

Comment:
Aren't we going to try using the `nui-icon` inputs to set the color?

Reply:
I guess we're not exposing those inputs to the wizard consumer. In this case, maybe we should avoid customizing the color in the example since we're not doing the full "puppy picture programming" implementation.